package.json

```diff
@@ -67,8 +67,6 @@
 		"autosize": "4.0.2",
 		"babel-loader": "8.0.5",
 		"babel-plugin-add-module-exports": "1.0.0",
```

Comment:
Can't remove that one quite yet :slightly_frowning_face: https://github.com/Automattic/wp-calypso/blob/e3618d2d93c4df194f2b8dcdef166b6c9b56cbe3/babel.config.js#L59